embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/EmbraceLogRecordExporter.kt

```diff
@@ -0,0 +1,17 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import io.opentelemetry.sdk.common.CompletableResultCode
+import io.opentelemetry.sdk.logs.data.LogRecordData
+import io.opentelemetry.sdk.logs.export.LogRecordExporter
+
+/**
+ * Exports the given [LogRecordData] to a [LogSink]
+ */
+internal class EmbraceLogRecordExporter(private val logSink: LogSink) : LogRecordExporter {
+    override fun export(logs: Collection<LogRecordData>): CompletableResultCode =
+        logSink.storeLogs(logs.toList())
+
+    override fun flush(): CompletableResultCode = CompletableResultCode.ofSuccess()
```

Comment:
Is this expected to be implemented later?

Reply:
I'm not sure what would need to be implemented here, since this exporter is stateless.